### DISCUSSION TITLE
[DISCUSSION] b-tensor encoding gradient table format

### DISCUSSION THREAD

**Discussion by mdesco** [dipy/dipy#3570] [1. Ideas & Feedback]
## Description
Moving an email thread to this DIPY issue. 
We need to discuss plan of action for b-tensor encoding gtab support in DIPY. 
@RafaelNH should take the lead or initiate a Doodle.

### Previous email discussion

Mail 1:
Hey Rafael and Dipy-ers,

I missed the lab meeting yesterday, sorry. Etienne talked to me today.

I’m cc-ing 2 students from my lab involved in b-tensor encoding and md-dmri dev in Sherbrooke. Both code in python. Philippe has worked on phase correction, complex data processing and now, b-tensor encoding spherical deconvolution. Alex has worked on DIAMOND and Magic-DIAMOND with my previous post-doc and Benoit Scherrer.

So, if you need:
1) Data, we have and we can share (linear, planar, spherical) on a clinical 3T Philips
  (Filip Scz. Has published a data paper recently also. Data is public)

2) Advise on bval/bvec encoding. We are used to deal with it. And we could be involved.

3) other…

Let us know how we can help.
Max, Phil, Alex

Response 1: 
Hi Maxime et al., 

Many thanks for your e-mail. Indeed, the data that Filip published might be a good starting point for our developments. At the moment, I was thinking to start implementing some techniques assuming that microstructural components are Gaussian and time-independent, so having the information of the full b-tensor in dipy's gradient table might be enough to start incorporating some reconstruction models into dipy. What do you think? 

Best,
Rafael

Response 2:
Hi Rafael,

Maybe we need a separate call for this. I'm not sure what you mean by "full b-tensor". For us, the gradient table is one thing and the "trajectory" another, which is coded as an analytical function when you know you are in the linear, planar or spherical case. So, aside from indicating that the bvals/bvecs are for a planar, spherical or linear b-tensor encoding, there is nothing else to be saved in the gradient table...

Maybe we are saying the samething with different terms. We could have a quick call between you, Phil, Alex and I if you want.

Max

**Comment by RafaelNH** [comment]
Thanks @mdesco! Yes, I will take the lead on this! I will go back to it ASAP!

**Comment by arokem** [comment]
Thanks for moving the conversation here! Worth also keeping an eye on
related discussions going on in the BIDS standard repo:
https://github.com/bids-standard/bids-specification/issues/349 and
https://github.com/bids-standard/bids-specification/pull/352.

For completeness, and for future reference, the link to the data is:
https://www.sciencedirect.com/science/article/pii/S2352340919305621?via%3Dihub

And related GitHub repo is here:
https://github.com/filip-szczepankiewicz/fwf_header_tools

**Comment by arokem** [comment]
Is this resolved by #2055?
